utility/doc/README.md

```diff
@@ -155,51 +155,3 @@ CheckTransaction(tx []byte) error
 GetProposalTransactions(proposer []byte, maxTransactionBytes int, lastBlockByzantineValidators [][]byte) (transactions [][]byte, err error)
 ApplyBlock(Height int64, proposer []byte, transactions [][]byte, lastBlockByzantineValidators [][]byte) (appHash []byte, err error)
 ```
-
-## How to build
```

Comment:
Was this intentionally moved from `README` to `PROTOCOLS`?

Reply:
No I'll move it back